backend/backend/events.py

```diff
@@ -219,18 +221,20 @@ def check_new_positions(output_stream, old_positions, new_positions):
     return new_positions
 
 
-def check_game_playing_status(output_stream, game):
+def check_game_playing_status(output_stream, game, current_game_state):
     """Check if the specified game's status is 'playing'.
 
     Arguments:
         game: The game whose status is being checked.
 
     """
-    if game.state == "playing":
+    if game.state != current_game_state and game.state == "playing":
```

Comment:
Minor point, probably: Would this be clearer as `if current_game_state == "waiting" and game.state == "playing":`?

In that vein, might make it clearer to call that variable `last_game_state` instead of `current_game_state`.